.github/docker/Dockerfile.alpine:3.16

```diff
@@ -0,0 +1,59 @@
+# Example of how to build this container (ran from pgx checkout directory):
+#   docker build --build-arg PG_MAJOR_VER=14 -f .github/docker/Dockerfile.alpine\:3.16 -t pgx-alpine .
```

Comment:
Thank you for documenting this in these!

Reply:
❤️ 

I love when I find the same documentation in other places. I figured I'd carry that forward :)